eva_cttv_pipeline/clinvar_xml_utils/clinvar_xml_utils/clinvar_xml_utils.py

```diff
@@ -1,13 +1,13 @@
 """Contains utilities and classes to parse the ClinVar XML and convert the records into internal representation via
-ClinVarDataset, ClinVarRecord, and ClinVarRecordMeasure classes."""
+ClinVarDataset, ClinVarRecord, ClinVarTrait, ClinVarRecordMeasure and ClinVarRecordMeasureHGVS classes."""
 
 import gzip
 import logging
 import re
 import xml.etree.ElementTree as ElementTree
 from functools import cached_property
 
-from eva_cttv_pipeline import clinvar_identifier_parsing
+from . import clinvar_identifier_parsing
```

Comment:
This is a bit weird, but it basically allows you to either use this as an independent module (where `.` == `clinvar_xml_utils`) or as a submodule of the pipeline (`.` == `eva_cttv_pipeline.clinvar_xml_utils`).

Reply:
Got it, thanks!